docs/sources/http_api/short_url.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Short URL HTTP API "
+description = "Grafana Short URL HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "shortUrl"]
+aliases = ["/docs/grafana/latest/http_api/short_url/"]
+type = "docs"
+[menu.docs]
+name = "Short URL"
+parent = "http_api"
++++
+
+# Short URL API
+
```

Comment:
Which Grafana release does this apply to?

Reply:
Probably 7.3